leaf-ui/rappid-extensions/AssignmentsTable.js

```diff
@@ -327,7 +332,9 @@ var IntentionRelationshipView = Backbone.View.extend({
     },
 
     render: function(){
-        this.$el.html(_.template($(this.template).html())(Object.assign(this.model, this.funcType, this.intentionName).toJSON()));
+        this.$el.html(_.template($(this.template).html())(this.model.toJSON()));
+        this.$('.intstart').text(this.intentionName + " : " + this.model.get('startTP'));
```

Comment:
Can you make these class names a bit more specific? aka "start-tp" / "func-type" etc